google-cloud-tools-plugin/src/com/google/gct/idea/debugger/CloudDebugProcessHandler.java

```diff
@@ -28,6 +32,16 @@
 
   public CloudDebugProcessHandler(CloudDebugProcess process) {
     myProcess = process;
+    if (myProcess != null && myProcess.getProcessState() != null) {
+      String userEmail = myProcess.getProcessState().getUserEmail();
+      if (userEmail != null) {
+        final CredentialedUser user = GoogleLogin.getInstance().getAllUsers().get(userEmail);
+        if (user.getGoogleLoginState() != null) {
```

Comment:
SGTM.

On Thu, Jan 28, 2016 at 11:51 AM, akerekes <notifications@github.com> wrote:

> In
> google-cloud-tools-plugin/src/com/google/gct/idea/debugger/CloudDebugProcessHandler.java
> <https://github.com/GoogleCloudPlatform/gcloud-intellij/pull/395#discussion_r51149651>
> :
>
> > @@ -28,6 +32,16 @@
> >
> >    public CloudDebugProcessHandler(CloudDebugProcess process) {
> >      myProcess = process;
> > +    if (myProcess != null && myProcess.getProcessState() != null) {
> > +      String userEmail = myProcess.getProcessState().getUserEmail();
> > +      if (userEmail != null) {
> > +        final CredentialedUser user = GoogleLogin.getInstance().getAllUsers().get(userEmail);
> > +        if (user.getGoogleLoginState() != null) {
>
> It can be null in case of incorrect API usage. We could log and error in
> IDEA that would show up as in internal error and if the user wants, he
> could submit it as a bug report.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleCloudPlatform/gcloud-intellij/pull/395/files#r51149651>
> .
>